playground/infrastructure/test_helper.py

```diff
@@ -211,9 +214,23 @@ def test__get_example():
         code="data",
         status=STATUS_UNSPECIFIED,
         tag=Tag(
-            "Name", "MEDIUM", "Description", "False", [""], "--option option", False, 1),
+            "Name", "MEDIUM", "{'kafka': {'topic': {'id': 'dataset', 'dataset': 'dataset'}}}", "{'dataset': {'location': 'GCS', 'format': 'json'}}", "Description", "False", [""], "--option option",
```

Comment:
I don't think it's a good idea to serialize json to a string tag and back. Could we just extend Example schema?